<a href="https://colab.research.google.com/github/jugranjeia/master-ddj-insper/blob/main/AN%C3%81LISE_EXPLORAT%C3%93RIA_DE_DADOS_II_TRABALHO_FINAL.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

#Analisando o custo do voto para governador no Rio de Janeiro
Grupo: Érico Monte, Julianna Granjeia, Luize Sampaio, Vinicius de Melo

Os arquivos usados podem ser baixados em: https://drive.google.com/drive/folders/1JAQV8XV6y4WnXheYUHVg-GGKAO75ZkZu

In [ ]:
import pandas

Após importar a biblioteca pandas, pedimos a leitura das depesesas de cada candidato ao governo do estado. Também fizemos a leitura da lista total dos candidatos. **Fonte:** DivulgaCand

In [ ]:
candidatos_RJ = pandas.read_csv('/content/Lista-Candidatos-Governador-RIO DE JANEIRO-2022.csv', sep = ';')
claudio_castro = pandas.read_csv('/content/Despesas-CLÁUDIO CASTRO-22-2022.csv', sep = ';')
cyro_garcia = pandas.read_csv('/content/Despesas-CYRO GARCIA-16-2022.csv', sep = ';')
eduardo_serra = pandas.read_csv('/content/Despesas-EDUARDO SERRA-21-2022.csv', sep = ';')
juliete = pandas.read_csv('/content/Despesas-JULIETE-80-2022.csv', sep = ';')
marcelo_freixo = pandas.read_csv('/content/Despesas-MARCELO FREIXO-40-2022.csv', sep = ';')
paulo_ganime = pandas.read_csv('/content/Despesas-PAULO GANIME -30-2022.csv', sep = ';')
rodrigo_neves = pandas.read_csv('/content/Despesas-RODRIGO NEVES-12-2022.csv', sep = ';')
wilson_witzel = pandas.read_csv('/content/Despesas-WILSON WITZEL-35-2022.csv', sep = ';')

In [ ]:
candidatos_RJ

Nome Completo       Nome Urna       Cargo  Número  \
0     CLÁUDIO BOMFIM DE CASTRO E SILVA  CLÁUDIO CASTRO  Governador      22   
1                          CYRO GARCIA     CYRO GARCIA  Governador      16   
2              EDUARDO GONÇALVES SERRA   EDUARDO SERRA  Governador      21   
3                JULIETE PANTOJA ALVES         JULIETE  Governador      80   
4                LUIZ EUGÊNIO HONORATO    LUIZ EUGÊNIO  Governador      29   
5               MARCELO RIBEIRO FREIXO  MARCELO FREIXO  Governador      40   
6  PAULO GUSTAVO GANIME ALVES TEIXEIRA   PAULO GANIME   Governador      30   
7                RODRIGO NEVES BARRETO   RODRIGO NEVES  Governador      12   
8                   WILSON JOSÉ WITZEL   WILSON WITZEL  Governador      35   

  Partido                         Coligação    Situação Situação Pós-Pleito  
0      PL            RIO UNIDO E MAIS FORTE    Deferido              Eleito  
1    PSTU                              PSTU    Deferido          Não eleito  
2     PCB                               PCB    Deferido          Não eleito  
3      UP                                UP    Deferido          Não eleito  
4     PCO                               PCO  Indeferido          Não eleito  
5     PSB               A VIDA VAI MELHORAR    Deferido          Não eleito  
6    NOVO                              NOVO    Deferido          Não eleito  
7     PDT  PRA MUDAR COM QUEM SABE GOVERNAR    Deferido          Não eleito  
8     PMB                               PMB  Indeferido          Não eleito

Começamos a análise pelo governador eleito, Cláudio Castro.

In [ ]:
claudio_castro.head()

CPF/CNPJ Fornecedor                                 Nome do Fornecedor  \
0         3.886047e+13              8EM7 INTELIGENCIA EM COMUNICACAO LTDA   
1         9.630990e+09                             AISLAN DE SOUZA COELHO   
2         4.672259e+13  ARROW AGENCIA DIGITAL, MARKETING E PARCERIAS LTDA   
3         4.672259e+13  ARROW AGENCIA DIGITAL, MARKETING E PARCERIAS LTDA   
4         3.230573e+13                  AUTO POSTO DE SERVICOS VILAR LTDA   

   CPF/CNPJ Doador Originário  Nome do Doador Originário        Data  \
0                         NaN                        NaN  16/08/2022   
1                         NaN                        NaN  16/08/2022   
2                         NaN                        NaN  16/08/2022   
3                         NaN                        NaN  16/08/2022   
4                         NaN                        NaN  01/10/2022   

  Especia do Recurso  Número do Documento  Recibo Eleitoral      Valor  
0         Financeiro                  1.0               NaN  200000.00  
1         Financeiro                  1.0               NaN    5118.00  
2         Financeiro                  1.0               NaN  120000.00  
3         Financeiro                  1.0               NaN  263000.00  
4         Financeiro                  1.0               NaN    3827.85

Para o CPF não aparecer como notação científica, fizemos uma alteração nas configurações do Pandas

In [ ]:
pandas.set_option('display.float_format', lambda x: '%.2f' % x)

In [ ]:
claudio_castro.head()

CPF/CNPJ Fornecedor                                 Nome do Fornecedor  \
0    38860472000197.00              8EM7 INTELIGENCIA EM COMUNICACAO LTDA   
1        9630989778.00                             AISLAN DE SOUZA COELHO   
2    46722592000156.00  ARROW AGENCIA DIGITAL, MARKETING E PARCERIAS LTDA   
3    46722592000156.00  ARROW AGENCIA DIGITAL, MARKETING E PARCERIAS LTDA   
4    32305732000186.00                  AUTO POSTO DE SERVICOS VILAR LTDA   

   CPF/CNPJ Doador Originário  Nome do Doador Originário        Data  \
0                         NaN                        NaN  16/08/2022   
1                         NaN                        NaN  16/08/2022   
2                         NaN                        NaN  16/08/2022   
3                         NaN                        NaN  16/08/2022   
4                         NaN                        NaN  01/10/2022   

  Especia do Recurso  Número do Documento  Recibo Eleitoral     Valor  
0         Financeiro                 1.00               NaN 200000.00  
1         Financeiro                 1.00               NaN   5118.00  
2         Financeiro                 1.00               NaN 120000.00  
3         Financeiro                 1.00               NaN 263000.00  
4         Financeiro                 1.00               NaN   3827.85

Somamos o valor dos gastos e chegamos ao resultado de **R$ 19.322.516,67**

In [ ]:
gastos_claudiocastro = claudio_castro['Valor'].sum()
gastos_claudiocastro

19322516.67

Importamos a base de votos totais do Rio de Janeiro pelo Google Drive, pois o Colab não aceitou o arquivo

In [ ]:
from google.colab import drive
drive.mount('/content/drive')

Drive already mounted at /content/drive; to attempt to forcibly remount, call drive.mount("/content/drive", force_remount=True).


In [ ]:
votos_RJ =  pandas.read_csv('/content/drive/MyDrive/Colab Notebooks/votacao_secao_2022_RJ.csv', sep = ';', encoding = "ISO-8859-1")

In [ ]:
votos_RJ

DT_GERACAO HH_GERACAO  ANO_ELEICAO  CD_TIPO_ELEICAO  \
0        01/11/2022   16:05:25         2022                2   
1        01/11/2022   16:05:25         2022                2   
2        01/11/2022   16:05:25         2022                2   
3        01/11/2022   16:05:25         2022                2   
4        01/11/2022   16:05:25         2022                2   
...             ...        ...          ...              ...   
6838265  01/11/2022   16:05:25         2022                2   
6838266  01/11/2022   16:05:25         2022                2   
6838267  01/11/2022   16:05:25         2022                2   
6838268  01/11/2022   16:05:25         2022                2   
6838269  01/11/2022   16:05:25         2022                2   

           NM_TIPO_ELEICAO  NR_TURNO  CD_ELEICAO  \
0        ELEIÇÃO ORDINÁRIA         1         546   
1        ELEIÇÃO ORDINÁRIA         1         546   
2        ELEIÇÃO ORDINÁRIA         1         546   
3        ELEIÇÃO ORDINÁRIA         1         546   
4        ELEIÇÃO ORDINÁRIA         1         546   
...                    ...       ...         ...   
6838265  ELEIÇÃO ORDINÁRIA         1         546   
6838266  ELEIÇÃO ORDINÁRIA         1         546   
6838267  ELEIÇÃO ORDINÁRIA         1         546   
6838268  ELEIÇÃO ORDINÁRIA         1         546   
6838269  ELEIÇÃO ORDINÁRIA         1         546   

                             DS_ELEICAO  DT_ELEICAO TP_ABRANGENCIA  ...  \
0        ELEIÇÕES GERAIS ESTADUAIS 2022  02/10/2022              E  ...   
1        ELEIÇÕES GERAIS ESTADUAIS 2022  02/10/2022              E  ...   
2        ELEIÇÕES GERAIS ESTADUAIS 2022  02/10/2022              E  ...   
3        ELEIÇÕES GERAIS ESTADUAIS 2022  02/10/2022              E  ...   
4        ELEIÇÕES GERAIS ESTADUAIS 2022  02/10/2022              E  ...   
...                                 ...         ...            ...  ...   
6838265  ELEIÇÕES GERAIS ESTADUAIS 2022  02/10/2022              E  ...   
6838266  ELEIÇÕES GERAIS ESTADUAIS 2022  02/10/2022              E  ...   
6838267  ELEIÇÕES GERAIS ESTADUAIS 2022  02/10/2022              E  ...   
6838268  ELEIÇÕES GERAIS ESTADUAIS 2022  02/10/2022              E  ...   
6838269  ELEIÇÕES GERAIS ESTADUAIS 2022  02/10/2022              E  ...   

        NR_SECAO CD_CARGO           DS_CARGO  NR_VOTAVEL  \
0             47        6   DEPUTADO FEDERAL        3607   
1            249        7  DEPUTADO ESTADUAL       44159   
2            136        7  DEPUTADO ESTADUAL       44159   
3            414        6   DEPUTADO FEDERAL        1933   
4             63        7  DEPUTADO ESTADUAL       44159   
...          ...      ...                ...         ...   
6838265       36        7  DEPUTADO ESTADUAL          22   
6838266       36        7  DEPUTADO ESTADUAL          29   
6838267       36        7  DEPUTADO ESTADUAL          30   
6838268       36        7  DEPUTADO ESTADUAL          44   
6838269       36        7  DEPUTADO ESTADUAL          55   

                         NM_VOTAVEL  QT_VOTOS  NR_LOCAL_VOTACAO  SQ_CANDIDATO  \
0            ROMULO GOMES DE MATTOS         2              1066  190001714931   
1         RONALDO DO CARMO ANQUIETA         1              1520  190001619411   
2         RONALDO DO CARMO ANQUIETA         1              1619  190001619411   
3        CARLOS HENRIQUE RIOS LEMOS         1              1880  190001609673   
4         RONALDO DO CARMO ANQUIETA         1              1180  190001619411   
...                             ...       ...               ...           ...   
6838265             Partido Liberal         6              1066            -3   
6838266   Partido da Causa Operária         1              1066            -3   
6838267                Partido Novo         3              1066            -3   
6838268                UNIÃO BRASIL         2              1066            -3   
6838269  Partido Social Democrático         1              1066            -3   

                    

Para facilitar a leitura, limitamos a quantidade de colunas dentro de um novo Data Frame e selecionamos apenas os candidatos a governador

In [ ]:
nova_votosRJ = votos_RJ.filter(['NR_TURNO','CD_MUNICIPIO','NM_MUNICIPIO','DS_CARGO','NR_VOTAVEL','NM_VOTAVEL','QT_VOTOS'], axis=1)

In [ ]:
governo_votosRJ = nova_votosRJ[nova_votosRJ['DS_CARGO'] == 'GOVERNADOR'] 

In [ ]:
governo_votosRJ

NR_TURNO  CD_MUNICIPIO    NM_MUNICIPIO    DS_CARGO  NR_VOTAVEL  \
769             1         60011  RIO DE JANEIRO  GOVERNADOR          30   
770             1         58122       QUEIMADOS  GOVERNADOR          30   
772             1         58491            MAGÉ  GOVERNADOR          30   
773             1         60011  RIO DE JANEIRO  GOVERNADOR          30   
774             1         60011  RIO DE JANEIRO  GOVERNADOR          30   
...           ...           ...             ...         ...         ...   
6768443         1         58637       NILÓPOLIS  GOVERNADOR          22   
6768444         1         60011  RIO DE JANEIRO  GOVERNADOR          22   
6768446         1         60011  RIO DE JANEIRO  GOVERNADOR          22   
6768447         1         58378        ITABORAÍ  GOVERNADOR          22   
6768448         1         58696     NOVA IGUAÇU  GOVERNADOR          22   

                                  NM_VOTAVEL  QT_VOTOS  
769      PAULO GUSTAVO GANIME ALVES TEIXEIRA        28  
770      PAULO GUSTAVO GANIME ALVES TEIXEIRA         4  
772      PAULO GUSTAVO GANIME ALVES TEIXEIRA         6  
773      PAULO GUSTAVO GANIME ALVES TEIXEIRA        20  
774      PAULO GUSTAVO GANIME ALVES TEIXEIRA        19  
...                                      ...       ...  
6768443     CLÁUDIO BOMFIM DE CASTRO E SILVA       137  
6768444     CLÁUDIO BOMFIM DE CASTRO E SILVA        69  
6768446     CLÁUDIO BOMFIM DE CASTRO E SILVA       148  
6768447     CLÁUDIO BOMFIM DE CASTRO E SILVA        99  
6768448     CLÁUDIO BOMFIM DE CASTRO E SILVA       165  

[246574 rows x 7 columns]

Somamos a quantidade de votos que Claudio Castro teve, usando seu número de urna como agredador. Um total de **4.930.288 votos**.

In [ ]:
votos_claudiocastro = governo_votosRJ.loc[governo_votosRJ['NR_VOTAVEL'] == 22, 'QT_VOTOS'].sum()
print(votos_claudiocastro)

4930288


Dividindo os gastos totais pela quantidade de votos, chegamos ao custo de cada voto individual em Cláudio Castro - cerca de **R$ 3,92**.

In [ ]:
custo_claudiocastro = gastos_claudiocastro / votos_claudiocastro
print(custo_claudiocastro)

3.9191456300321605


Repetimos a operação com os outros candidatos: Cyro Garcia, Eduardo Serra, Juliete, Marcelo Freixo, Paulo Ganime, Wilson Witzel, Rodrigo Neves.

In [ ]:
gastos_cyrogarcia = cyro_garcia['Valor'].sum()
votos_cyrogarcia = governo_votosRJ.loc[governo_votosRJ['NR_VOTAVEL'] == 16, 'QT_VOTOS'].sum()
custo_cyrogarcia = gastos_cyrogarcia / votos_cyrogarcia
print(custo_cyrogarcia)

14.420993110002376


In [ ]:
gastos_eduardoserra = eduardo_serra['Valor'].sum()
votos_eduardoserra = governo_votosRJ.loc[governo_votosRJ['NR_VOTAVEL'] == 21, 'QT_VOTOS'].sum()
custo_eduardoserra = gastos_eduardoserra / votos_eduardoserra
print(custo_eduardoserra)

9.208938444526353


In [ ]:
gastos_juliete = juliete['Valor'].sum()
votos_juliete = governo_votosRJ.loc[governo_votosRJ['NR_VOTAVEL'] == 80, 'QT_VOTOS'].sum()
custo_juliete = gastos_juliete / votos_juliete
print(custo_juliete)

0.7455361322410766


In [ ]:
gastos_marcelofreixo = marcelo_freixo['Valor'].sum()
votos_marcelofreixo = governo_votosRJ.loc[governo_votosRJ['NR_VOTAVEL'] == 40, 'QT_VOTOS'].sum()
custo_marcelofreixo = gastos_marcelofreixo / votos_marcelofreixo
print(custo_marcelofreixo)

4.91914857147824


In [ ]:
gastos_pauloganime = paulo_ganime['Valor'].sum()
votos_pauloganime = governo_votosRJ.loc[governo_votosRJ['NR_VOTAVEL'] == 30, 'QT_VOTOS'].sum()
custo_pauloganime = gastos_pauloganime / votos_pauloganime
print(custo_pauloganime)

2.694796587397555


In [ ]:
gastos_rodrigoneves = rodrigo_neves['Valor'].sum()
votos_rodrigoneves = governo_votosRJ.loc[governo_votosRJ['NR_VOTAVEL'] == 12, 'QT_VOTOS'].sum()
custo_rodrigoneves = gastos_rodrigoneves / votos_rodrigoneves
print(custo_rodrigoneves)

20.551338527512637


In [ ]:
gastos_wilsonwitzel = wilson_witzel['Valor'].sum()
votos_wilsonwitzel = governo_votosRJ.loc[governo_votosRJ['NR_VOTAVEL'] == 35, 'QT_VOTOS'].sum()
custo_wilsonwitzel = gastos_wilsonwitzel / votos_wilsonwitzel
print(custo_wilsonwitzel)

0.36999182338511855


##Chegamos a conclusão que:

* **Rodrigo Neves (PDT)** e **Cyro Garcia (PSTU)** foram os candidatos que mais gastaram na campanha e menos tiveram retorno;
* A campanha de **Juliete (UP)** foi a mais barata dentre as deferidas;
* **Wilson Witzel (PMB)** aparece com o menor custo por voto, mas sua campanha foi indeferida pela lei da Ficha Limpa.
* Comparado aos demais candidatos, **Claudio Castro (PL)** está entre os mais econômicos no Rio de Janeiro.



##Comparando com outros eleitos da região Sudeste

In [ ]:
votos_sp = pandas.read_csv('/content/drive/MyDrive/Colab Notebooks/votacao_candidato_munzona_2022_SP.csv', sep = ';', encoding = "ISO-8859-1")
votos_mg = pandas.read_csv('/content/drive/MyDrive/Colab Notebooks/votacao_candidato_munzona_2022_MG.csv', sep = ';', encoding = "ISO-8859-1")
votos_es = pandas.read_csv('/content/drive/MyDrive/Colab Notebooks/votacao_candidato_munzona_2022_ES.csv', sep = ';', encoding = "ISO-8859-1")

In [ ]:
votos_tarcisio = votos_sp.loc[votos_sp['NR_CANDIDATO'] == 10, 'QT_VOTOS_NOMINAIS_VALIDOS'].sum()
votos_zema = votos_mg.loc[votos_mg['NR_CANDIDATO'] == 30, 'QT_VOTOS_NOMINAIS_VALIDOS'].sum()
votos_casagrande = votos_es.loc[votos_es['NR_CANDIDATO'] == 40, 'QT_VOTOS_NOMINAIS_VALIDOS'].sum()

In [ ]:
tarcisio = pandas.read_csv('/content/Despesas-TARCÍSIO-10-2022.csv', sep = ';')
zema = pandas.read_csv('/content/Despesas-ZEMA-30-2022.csv', sep = ';')
casagrande = pandas.read_csv('/content/Despesas-RENATO CASAGRANDE-40-2022.csv', sep = ';')

In [ ]:
gastos_tarcisio = tarcisio['Valor'].sum()
gastos_zema = zema['Valor'].sum()
gastos_casagrande = casagrande['Valor'].sum()

In [ ]:
custo_tarcisio = gastos_tarcisio / votos_tarcisio
print('Tarcísio Freitas - SP -', custo_tarcisio)
custo_zema = gastos_zema / votos_zema
print('Romeu Zema - MG -', custo_zema)
custo_casagrande = gastos_casagrande / votos_casagrande
print('Renato Casagrande - ES -', custo_casagrande)

Tarcísio Freitas - SP - 1.6021844245500014
Romeu Zema - MG - 2.914954144443117
Renato Casagrande - ES - 4.698371919141131


##Chegamos a conclusão que:

* **Tarcísio Freitas (Republicanos)** foi o candidato do Sudeste que menos gastou para conquistar um voto - **R$ 1,60**;
* **Romeu Zema (Novo)** ficou em segundo lugar, com gasto médio de **2,91 reais** por voto;
* **Cláudio Castro (PL)** ficou em terceiro lugar, com gasto médio **3,92 reais** por voto;
* **Renato Casagrande (PSB)** fo que mais gastou, com custo de **4,70 reais** por voto.

**Acusado de abuso de poder, Claudio Castro teve um dos votos mais baratos entre os candidatos a governador do Rio**

O voto no governador reeleito do Rio de Janeiro, Cláudio Castro (PL), custou apenas  R$ 3,90. Entre os oito candidatos que disputaram a eleição 2022 no estado, Castro foi o terceiro político com o voto mais barato. No entanto, sua reeleição está ameaçada.

A Procuradoria Regional Eleitoral do Rio pediu, no dia 14 de dezembro, a inelegibilidade e a cassação da chapa eleita do governador. Ele é acusado de abuso de poder político e econômico e de conduta vedada a agente público no escândalo das contratações secretas em ano eleitoral para a Fundação Centro Estadual de Estatística, Pesquisa e Formação de Servidores Públicos do Rio (Ceperj) e para programas da Universidade do Estado do Rio de Janeiro (Uerj).

Segundo o jornal O Globo, a assessoria do governador Cláudio Castro e do vice na chapa, Thiago Pampolha (União Brasil), afirmou que a chapa "vai comprovar a conduta idônea" e que ambos estão "seguros de que todos os pontos levantados serão respondidos".

 
**METODOLOGIA**
 
O cálculo do preço do voto foi feito a partir dos dados disponibilizados pelo TSE (Tribunal Superior Eleitoral) e com uso da biblioteca Pandas para análise de dados com Python. Primeiramente, analisamos os gastos dos candidatos ao governo do estado do Rio de Janeiro durante a campanha de 2022 e cruzamos com o total de votos.

Castro gastou R 19.322.516,67 e obteve 4.930.288 votos (58,6%) . Dividindo os gastos totais pela quantidade de votos, chegamos ao custo de cada voto individual em Castro - R$ 3,92.

É preciso considerar que quanto maior a quantidade de votos que um candidato recebe, mais diluído será o valor gasto durante sua campanha. Logo, o candidato eleito terá um valor de voto menor.

A operação usada para chegarmos a esses números está detalhada no arquivo collab anexo. Repetimos o mesmo procedimento com os outros candidatos: Cyro Garcia (PSTU), Eduardo Serra (PCB), Juliete Pantoja (UP), Marcelo Freixo (PSB), Paulo Ganime (NOVO), Wilson Witzel (PMB) e Rodrigo Neves (PDT).
 
**Quanto custou cada voto?**

Rodrigo Neves (PDT) - R$ 20.5 (8%)

Cyro Garcia (PSTU) - R$ 14.4 (0,15%)

Eduardo Serra (PCB) - R$ 9.2 (0,13%)

Marcelo Freixo (PSB) - R$ 4.9 (27,38%)

Claudio Castro (PL) - R$ 3,9 (58,67%)

Paulo Ganime (NOVO) - R$ 2.6 (5,31%)

Juliete Pantoja (UP) - R$ 0.7 (0,33%)

Wilson Witzel (PMB) - R$ 0.3 (candidatura indeferida)
 
**CURIOSIDADES**

Rodrigo Neves (PDT), ex-prefeito de Niteroi, que teve o ex-presidente da Ordem dos Advogados do Brasil (OAB) Felipe Santa Cruz (PSD) como vice na chapa, foi o candidato com o voto mais caro. 

Ainda em outubro, a Justiça do Rio de Janeiro tornou Neves réu em um processo sobre corrupção em contratos assinados na época em que ele era prefeito de Niterói, na Região Metropolitana. O Ministério Público Federal (MPF) afirma que ele direcionou licitações e fraudou editais com a ajuda de conselheiros do Tribunal de Contas do estado.

Já o voto mais barato, entre as candidaturas deferidas, foi de Juliete (UP), a única candidata mulher no estado.

Wilson Witzel (PMB), que sofreu um impeachment do governo do Rio em 2021 - quando seu vice, Castro, assumiu - apareceu com o menor custo por voto, mas sua campanha foi indeferida.

Também comparamos os números dos candidatos do Rio de Janeiro com os de outros eleitos da região Sudeste e descobrimos que:

Tarcísio Freitas (Republicanos), governador eleito em São Paulo, foi o candidato do Sudeste que menos gastou para conquistar um voto - R$ 1,60;

Romeu Zema (Novo), reeleito em Minas Gerais, ficou em segundo lugar, com gasto médio de R$ 2,91 por voto;

Cláudio Castro (PL), no Rio, ficou em terceiro lugar, com gasto médio R$ 3,92 por voto;

Renato Casagrande (PSB), governador reeleito do Espírito Santo, foi quem mais gastou, com custo de R$ 4,70 por voto.
 

Grupo: Érico Monte, Julianna Granjeia, Luize Sampaio, Vinicius de Melo
